spec/pr_size_checker_spec.rb

```diff
@@ -0,0 +1,194 @@
+          deleted_file1 = 'src/java/org/Fire.kt'
+          deleted_file2 = 'BlackCat.kt'
+          deleted_test_file = 'src/test/java/org/magic/Power.java'
+          deleted_strings = 'src/main/res/values-de/strings.xml'
+
+          allow(@plugin.git).to receive(:added_files).and_return([added_config, added_file])
+          allow(@plugin.git).to receive(:modified_files).and_return([modified_file1, modified_file2, added_test_file, modified_strings])
+          allow(@plugin.git).to receive(:deleted_files).and_return([deleted_file1, deleted_test_file, deleted_strings, deleted_file2])
+
+          allow(@plugin.git).to receive(:info_for_file).with(added_test_file).and_return({ insertions: 201 })
+          allow(@plugin.git).to receive(:info_for_file).with(added_config).and_return({ insertions: 311 })
+          allow(@plugin.git).to receive(:info_for_file).with(added_file).and_return({ insertions: 13 })
+          allow(@plugin.git).to receive(:info_for_file).with(modified_file1).and_return({ insertions: 127, deletions: 159 })
+          allow(@plugin.git).to receive(:info_for_file).with(modified_file2).and_return({ insertions: 43, deletions: 37 })
+          allow(@plugin.git).to receive(:info_for_file).with(modified_strings).and_return({ insertions: 432, deletions: 297 })
+          allow(@plugin.git).to receive(:info_for_file).with(deleted_file1).and_return({ deletions: 246 })
+          allow(@plugin.git).to receive(:info_for_file).with(deleted_file2).and_return({ deletions: 493 })
+          allow(@plugin.git).to receive(:info_for_file).with(deleted_test_file).and_return({ deletions: 222 })
+          allow(@plugin.git).to receive(:info_for_file).with(deleted_strings).and_return({ deletions: 593 })
+        end
+      end
+
+      context 'when checking a PR body size' do
+        it 'returns a warning when using default parameters in a PR that has a smaller body text length than the default minimum' do
+          allow(@plugin.github).to receive(:pr_body).and_return('pr body')
+
+          @plugin.check_pr_body
+
+          expect(@dangerfile.status_report[:warnings]).to eq ['Please provide a summary of the changes in the Pull Request description']
+        end
+
+        it 'does nothing when using default parameters in a PR that has a bigger pr body text length than the default minimum' do
+          allow(@plugin.github).to receive(:pr_body).and_return('some test pr body')
+
+          @plugin.check_pr_body
+
+          expect(@dangerfile.status_report[:warnings]).to be_empty
+        end
+
+        it 'returns an error when using default parameters in a PR that has an equal pr body text length than the default minimum' do
+          allow(@plugin.github).to receive(:pr_body).and_return('some test-')
+
+          @plugin.check_pr_body
+
+          expect(@dangerfile.status_report[:warnings]).to eq ['Please provide a summary of the changes in the Pull Request description']
```

Comment:
```suggestion
        it 'returns a warning when using default parameters in a PR that has an equal pr body text length than the default minimum' do
          allow(@plugin.github).to receive(:pr_body).and_return('some test-')

          @plugin.check_pr_body

          expect(@dangerfile.status_report[:warnings]).to eq ['Please provide a summary of the changes in the Pull Request description']
```

Or, if my nitpick about reporting vs returning resonates,

```suggestion
        it 'reports a warning when using default parameters in a PR that has an equal pr body text length than the default minimum' do
          allow(@plugin.github).to receive(:pr_body).and_return('some test-')

          @plugin.check_pr_body

          expect(@dangerfile.status_report[:warnings]).to eq ['Please provide a summary of the changes in the Pull Request description']
```